### PR TITLE
Add admin

### PR DESCRIPTION
**Pull request by penovicp**

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (SherrySeibel) [inline app/admin/dashboard.rb]: These comments can be deleted

File: app/admin/user.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  permit_params do
+    params = form_fields
+    params += [:password, :password_confirmation]
+    params
+  end
+
+  batch_action :approve do |ids|
+    set_batch_approved.call(ids, flash, true)
+    redirect_to(:back)
```
Comment (SherrySeibel) [inline app/admin/user.rb]: The argument after `redirect_to` does not actually require parentheses. They should be removed for syntax consistency throughout our app.

File: db/migrate/20150415180759_devise_create_admin_users.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+class DeviseCreateAdminUsers < ActiveRecord::Migration
+  def migrate(direction)
+    super
+    # Create a default user
+    AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if direction == :up
```
Comment (SherrySeibel) [inline db/migrate/20150415180759_devise_create_admin_users.rb]: This line should be broken up so it is not over 80 characters.
Maybe something like this?

```ruby
if direction == :up
  dminUser.create!(
  email: 'admin@example.com',
  password: 'password', 
  password_confirmation: 'password'
)
end
```

Comment (SherrySeibel) [inline db/migrate/20150415180759_devise_create_admin_users.rb]: Nevermind. this is a migration. Sorry!

File: spec/controllers/admin/users_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+require 'spec_helper'
```
Comment (SherrySeibel) [inline spec/controllers/admin/users_controller_spec.rb]: You only need to require the `rails_helper`

**Comment by iblazevic** [comment]
lgtm

**Comment by SherrySeibel** [comment]
:+1:

**Pull request merged**